playbooks/02 - Use Case - Asset Change Activity/Add Cyber Asset.json

```diff
@@ -5289,27 +5291,238 @@
                 "step_variables": []
             },
             "status": null,
-            "top": "840",
-            "left": "125",
+            "top": "1110",
+            "left": "475",
             "stepType": "\/api\/3\/workflow_step_types\/dc6ac63d-c5a5-472f-9eb4-6b18473a98b8",
             "group": null,
             "uuid": "890ce319-249e-477b-9523-1de9ca1c30f7"
         },
+        {
+            "@type": "WorkflowStep",
+            "name": "Comment Approver Approval To Implement Task Complete",
```

Comment:
Didn't get this playbook name. Why description is empty?

Reply:
Incorporated